athlinks-schema.ttl

```diff
@@ -174,4 +193,107 @@ athlinks:chipTime
 athlinks:agePercentage
   a owl:DataTypeProperty ;
   rdfs:domain athlinks:IntervalResult;
-  rdfs:range schema:Decimal .
+  rdfs:range xsd:decimal .
+
+athlinks:isHidden
+  a owl:DataTypeProperty ;
+  rdfs:domain athlinks:AthleticRace;
+  rdfs:range xsd:boolean .
+
+athlinks:isVisible
+  a owl:DataTypeProperty ;
+  rdfs:domain athlinks:AthleticRace;
+  rdfs:range xsd:boolean .
```

Comment:
Aren't isHidden and isVisible the same thing in inverse?
Can we use just one?
Missing label and comment.